ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -44,7 +44,7 @@ public enum TableViewCellAccessoryType: Int {
         case .detailButton:
             return UIColor(dynamicColor: tokenSet[.accessoryDetailButtonColor].dynamicColor)
         case .checkmark:
-            return UIColor(dynamicColor: fluentTheme.aliasTokens.brandColors[.primary])
+            return UIColor(dynamicColor: fluentTheme.aliasTokens.colors[.brandForeground1])
```

Comment:
Do we know why this isn't in the table view cell tokens?

Reply:
I do not know why! I think that's outside the scope of this PR though. I can file a bug to track this.